### PR TITLE
new: added links to Swift Forums on Support/nav bar

### PR DESCRIPTION
**Pull request by KyeMaloy97**

### CODE REVIEW

**Comment by KyeMaloy97** [comment]
screen shot 2018-05-11 at 11 06 19 am

**Comment by helenmasters** [comment]
@ianpartridge please can you check that you're happy with the wording which is associated with the Swift Forums.

File: _includes/header/header-en.html [hunk 1]
```diff
@@ -37,6 +37,6 @@
         <section id="social">
 		<a href="http://slack.kitura.io/" target="_blank" title="Join the Kitura community on Slack"><img  class="slack-icon" src="/assets/slack.png" alt="Slack icon" width="22" height="22"/></a>
 		<a href="http://github.com/IBM-Swift/Kitura" target="_blank" title="Contribute to Kitura on GitHub"><img class="github-icon" src="/assets/github.png" alt="GitHub icon" width="22" height="22"/></a>
-		<a href="https://stackoverflow.com/questions/tagged/kitura" target="_blank" title="Ask a Kitura question on StackOverflow"><img class="stackoverflow-icon"  src="/assets/stackoverflow.png" alt="StackOverflow icon" width="22" height="22"/></a>
+		<a href="https://forums.swift.org/c/related-projects/kitura" target="_blank" title="Post to the Kitura topic on the Swift Forums"><img class="swiftforum-icon"  src="/assets/swift-forums.png" alt="SwiftForums icon" width="22" height="22"/></a>
```
Comment (ianpartridge) [inline _includes/header/header-en.html]: topic -> category

File: en/support/support.md [hunk 2]
```diff
@@ -25,7 +25,11 @@ Open an issue or get involved and raise a pull request on [GitHub](https://githu
 
 ## Slack
 
-Join the Kitura community on [Slack](http://swift-at-ibm-slack.mybluemix.net/).
+Join the Kitura community on [Slack](http://slack.kitura.io/).
+
+## Swift Forums
+
+Post to the Kitura topic, as well as browse or submit new proposals for the language, all on the [Swift Forums](https://forums.swift.org/c/related-projects/kitura).
```
Comment (ianpartridge) [inline en/support/support.md]: topic -> category

**Comment by KyeMaloy97** [comment]
Will open two separate PRs as the nav bar icon requires a new icon which in turn needs legal approval and permission.

**Pull request closed without merging**